src/pages/Admin/Admin.tsx

```diff
@@ -427,7 +424,7 @@ const Admin: React.FC = () => {
 
   const { isAdminUser, isLoading: isAdminUserLoading } = useIsAdminUser(userId);
 
-  const venuesOwnedByUserQuery = useMemo<ReduxFirestoreQuerySetting>(
+  const venuesOwnedByUserQuery = useMemo<SparkleRFQuery>(
```

Comment:
[NB, Future Improvement]
Does eslint let us warn/error on things like this? `useMemo<SparkleRFQuery>` (may just make more sense to fix all cases of it instead. But would be interesting to know.